public/js/databases_tree.js

```diff
@@ -76,22 +79,29 @@ export default class extends Databases {
         // JSX.
         return (
             <div className={columnClass} key={'DB_' + category}>
-                <div className='panel panel-default' id='database_list'>
-                    <div className='panel-heading'>
-                        <h4 style={{ display: 'inline' }}>{panelTitle}</h4> &nbsp;&nbsp;
+                <div>
+                    <div className="border-b border-seqorange mb-2" id="database_list">
+                        <h4 style={{ display: 'inline' }} className="font-medium">{panelTitle}</h4>
                         {
                             this.renderDatabaseSearch(category)
                         }
-                        <button type='button' className={toggleClass} disabled={toggleDisabled} style={{ display: 'none' }}
-                            onClick={function () { this.handleToggle(toggleState, category); }.bind(this)}>
+                        <button
+                            type="button"
+                            className={toggleClass}
+                            disabled={toggleDisabled}
+                            style={{ display: 'none' }}
```

Comment:
Do we still need to use inline styles here and in other places? Can we use tailwind classes instead?

Reply:
updated